.circleci/config.yml

```diff
@@ -24,7 +21,7 @@ jobs:
       - checkout
       - run: apk update && apk add ca-certificates make
       - restore_cache:
-          <<: *binary-cache-key
+          key: vendor-cache-{{ .Environment.CIRCLE_PROJECT_REPONAME }}-{{ checksum "Gopkg.lock" }}-v1
```

Comment:
🏄 possibly worth creating a new anchor for the actual key string and using that, to save declaring the full key string twice.